src/main/resources/assets/gilly7ce-carpet-addons/lang/en_us.json

```diff
@@ -5,5 +5,6 @@
   "carpet.rule.netheritePickaxeInstantMineCobblestone.desc": "A netherite pickaxe with efficiency V combined with the haste II status effect will instant mine cobblestone type blocks.",
   "carpet.rule.netheritePickaxeInstantMineDeepslate.desc": "A netherite pickaxe with efficiency V combined with the haste II status effect will instant mine deepslate type blocks.",
   "carpet.rule.netheritePickaxeInstantMineEndStone.desc": "A netherite pickaxe with efficiency V combined with the haste II status effect will instant mine end stone type blocks.",
-  "carpet.rule.netheritePickaxeInstantMineNetherBricks.desc": "A netherite pickaxe with efficiency V combined with the haste II status effect will instant mine nether brick type blocks."
+  "carpet.rule.netheritePickaxeInstantMineNetherBricks.desc": "A netherite pickaxe with efficiency V combined with the haste II status effect will instant mine nether brick type blocks.",
+  "carpet.rule.dropEyeFromFrame.desc": "End portal frame with eye inside of it will drop eye and change its state to not have eye when right clicked by a player"
```

Comment:
Change description to be "A full end portal frame will drop an eye of ender when right clicked by a player, turning into an empty end portal frame in the process".

Terminology from the [minecraft wiki](https://minecraft.fandom.com/wiki/End_Portal_Frame).

Reply:
Sorry, realised my description doesn't described the portal state. Description should be:
"A full end portal frame will drop an eye of ender when right clicked by a player, turning into an empty end portal frame in the process. Any connecting end portals will break."